tests/devices/test_local_decoder_padding.py

```diff
@@ -0,0 +1,89 @@
+from roborock.protocol import create_local_decoder, create_local_encoder
+from roborock.roborock_message import RoborockMessage, RoborockMessageProtocol
+
+TEST_LOCAL_KEY = "local_key"
+
+
+def test_decoder_clean_message():
+    encoder = create_local_encoder(TEST_LOCAL_KEY)
+    decoder = create_local_decoder(TEST_LOCAL_KEY)
+
+    msg = RoborockMessage(
+        protocol=RoborockMessageProtocol.RPC_REQUEST,
+        payload=b"test_payload",
+        version=b"1.0",
+        seq=1,
+        random=123,
+    )
+    encoded = encoder(msg)
+
+    decoded = decoder(encoded)
+    assert len(decoded) == 1
+    assert decoded[0].payload == b"test_payload"
```

Comment:
I believe you can write the first three tests as one test like this:

```suggestion
@pytest.mark.parametrize(
    ("garbage"),
    [
        b"",
        b"\x00\x00\x05\xa1"
        b"\x00\x00\x05\xa1\xff\xff"
    ],
)
def test_decoder_clean_message(garbage: bytes):
    encoder = create_local_encoder(TEST_LOCAL_KEY)
    decoder = create_local_decoder(TEST_LOCAL_KEY)

    msg = RoborockMessage(
        protocol=RoborockMessageProtocol.RPC_REQUEST,
        payload=b"test_payload",
        version=b"1.0",
        seq=1,
        random=123,
    )
    encoded = encoder(msg)

    decoded = decoder(garbage + encoded)
    assert len(decoded) == 1
    assert decoded[0].payload == b"test_payload"
```